data/data.go

```diff
@@ -0,0 +1,39 @@
+package data
+
+import (
+	packr "github.com/gobuffalo/packr/v2"
+	yaml "gopkg.in/yaml.v2"
```

Comment:
`gopkg.in` uses fallbacks to determine the git repo for a given URL. There has been at least one case in the past where a user moved their github repo and someone else set up their own fork as the target for the `gopkg.in` url. To avoid this security problem I would suggest using `github.com/go-yaml/yaml` as the import url. Now that go modules exist there is no reason to use `gopkg.in` urls.